index.html

```diff
@@ -165,8 +165,32 @@ <h4 class="connect">
             <li>Frameworks<span class="angle"><img src="images/angle-right.svg" alt=""></span> </li>
             <img src="images/Separator Bottom.svg" alt="" srcset="">
             <li>Skills<span class="angle"><img src="images/angle-right.svg" alt=""></span> </li>
-            <img src="images/Separator Bottom.svg" alt="" srcset="">
+            <img class="last-line" src="images/Separator Bottom.svg" alt="" srcset="">
         </ul>
     </section>
+    <section class="contact-section">
+        <img  class="c-eli1" src="images/contact-Ellipse1.svg" alt="">
+        <img class="c-eli2" src="images/contact-Rectangle1.svg" alt="">
+        <img class="c-eli3" src="images/contact-Ellipse2.svg" alt="">
+        <h3 class="contact-title"> Contact me</h3>
+        <div >
+            <p class="contact-text">If you have an application you are interested in developing, a feature that you need  built or a project that needs coding. 
+                I’d love to help with it!
+            </p> 
+        </div>
+        <form action="https://formspree.io/f/xbjezrzn" method="POST">
+            <ul>
+                <li><input name="name" type="text" maxLength="30" placeholder="LoremIpsum"></li>
+                <li><input type="email" name="email" id="email" placeholder="loremipsum@gmail.com"></li>
+                <li><textarea maxLength="500" name="message" id="" cols="30" rows="10" placeholder="Write your message here...."></textarea></li>
```

Comment:
### Required Changes
- You might want to remove the ul tag in the form.  This is because ul tag is mostly used as a bullet when you are listing something on your page but in this case, the input element alone is enough and they do not need to have a bullet attached to it.
- Kindly replace the lorem text in your placeholder with a proper name of the input field so that the user will be able to provide their correct information